coordinator/main.go

```diff
@@ -103,20 +112,30 @@ func run() (retErr error) {
 			return fmt.Errorf("failed to listen: %w", err)
 		}
 		if err := grpcServer.Serve(lis); err != nil {
+			logger.Error("serving Coordinator API", "err", err)
 			return fmt.Errorf("serving Coordinator API: %w", err)
 		}
 		return nil
 	})
 
 	eg.Go(func() error {
-		meshAPI := newMeshAPIServer(meshAuth, meshAuth, promRegistry, serverMetrics, logger)
 		logger.Info("Coordinator mesh API listening")
 		if err := meshAPI.Serve(net.JoinHostPort("0.0.0.0", meshapi.Port)); err != nil {
+			logger.Error("serving mesh API", "err", err)
```

Comment:
What are the extra logs for here? Shouldn't the returned errors be recognized by `.Wait()` anyway?

Reply:
Wait will only return the first error.